conjureup/controllers/newcloud/common.py

```diff
@@ -148,34 +154,27 @@ def setup_conjureup0_network():
                     out.stderr.decode()))
 
 
-def setup_lxdbr0_network():
-    """ This attempts to setup LXD networking if not available
+def setup_bridge_network(iface):
+    """ Sets up our main network bridge to be used with Localhost deployments
     """
     try:
-        utils.run('lxc network show lxdbr0', shell=True, check=True,
+        utils.run('lxc network show conjureup1', shell=True, check=True,
                   stdout=DEVNULL, stderr=DEVNULL)
     except CalledProcessError:
-        out = utils.run_script('lxc network create lxdbr0 '
-                               'ipv4.address=10.0.8.1/24 '
+        out = utils.run_script('lxc network create conjureup1 '
+                               'ipv4.address=10.100.0.1/24 '
```

Comment:
To not collide with users previous lxdbr0 setup via conjure-up. We don't unset lxdbr0 during snap install so we just pick another subnet here.